### DISCUSSION TITLE
TypeError: null is not an object (evaluating 'sl.name')

### DISCUSSION THREAD

**Discussion by cargeo** [ingrammicro/puzzle-publisher#85] [General]
When exporting one or multiple artboards via "Export selected to HTML", I get the following error:
`TypeError: null is not an object (evaluating 'sl.name')`

Additionally some artboards get all their symbols detached. (This seems a side effect with every error of this plugin)

I get this error only occasionally and I don't know what it causes. The same artboards/document who throw this error are sometimes totally OK later that day.

Can you help?

**Comment by MaxBazarov** [comment]
Try to  set "Enable Element Inspector" on in Configure Export dialog.

**Comment by cargeo** [comment]
Export works with "Enable Element Inspector" enabled. But I'd like to disable/hide it for my presentation.
